src/components/structures/RoomView.js

```diff
@@ -1193,8 +1197,8 @@ module.exports = React.createClass({
                             <div className="mx_RoomView_tabCompleteImage">...</div>
                             <div className="mx_RoomView_tabCompleteWrapper">
                                 <TabCompleteBar entries={this.tabComplete.peek(6)} />
-                                <div className="mx_RoomView_tabCompleteEol">
-                                    <img src="img/eol.svg" width="22" height="16" alt="->|"/>
+                                <div className="mx_RoomView_tabCompleteEol" title="->|">
+                                    <object onLoad={ this.onSvgLoad } className="mx_Svg" type="image/svg+xml" data="img/eol.svg" width="22" height="16"/>
```

Comment:
It would be great to factor out this whole <div> structure, complete with its onSvgLoad, to a separate react component so you don't have to c&p the idiom 150 times.